### PR TITLE
feat: update for split catwalk & whiskers repos

### PR DESCRIPTION
**Pull request by backwardspy**

### CODE REVIEW

File: Formula/catwalk.rb [hunk 1]
```diff
@@ -1,15 +1,10 @@
 class Catwalk < Formula
   desc "Soothing preview image generator for the high-spirited!"
-  homepage "https://github.com/catppuccin/toolbox/tree/main/catwalk"
+  homepage "https://github.com/catppuccin/catwalk"
   url "https://github.com/catppuccin/toolbox/archive/refs/tags/catwalk-v1.3.1.tar.gz"
```
Comment (sgoudham) [inline Formula/catwalk.rb:4]:
```suggestion
  url "https://github.com/catppuccin/catwalk/archive/refs/tags/v1.3.1.tar.gz"
```

File: Formula/whiskers.rb [hunk 2]
```diff
@@ -1,15 +1,10 @@
 class Whiskers < Formula
   desc "Soothing port creation tool for the high-spirited!"
-  homepage "https://github.com/catppuccin/toolbox/tree/main/whiskers"
+  homepage "https://github.com/catppuccin/whiskers"
   url "https://github.com/catppuccin/toolbox/archive/refs/tags/whiskers-v2.3.0.tar.gz"
```
Comment (sgoudham) [inline Formula/whiskers.rb:4]:
```suggestion
  url "https://github.com/catppuccin/whiskers/archive/refs/tags/v2.3.0.tar.gz"
```

**Review by sgoudham — CHANGES_REQUESTED** [review]
If not updating the URLs was intentional, dismiss this review

**Comment by backwardspy** [comment]
> If not updating the URLs was intentional, dismiss this review

my thinking was renovate would update the url + hash in a similar way to `brew bump-formula-pr` does

**Comment by backwardspy** [comment]
my thinking was wrong #29

**Pull request merged**